setup.py

```diff
@@ -1,31 +1,31 @@
-from setuptools import find_packages
 from distutils.core import setup
 
+from setuptools import find_packages
+
 
 def readme():
-    with open('README.rst') as f:
+    with open("README.rst") as f:
         return f.read()
 
+
 setup(
-    name='onesignal',
-    packages=find_packages(exclude=['tests*']),
-    version='0.1.3',
-    description='A Python wrapper for OneSignal API (http://onesignal.com)',
+    name="onesignal",
+    packages=find_packages(exclude=["tests*"]),
+    version="0.2.0",
+    description="A Python wrapper for OneSignal API (http://onesignal.com)",
     long_description=readme(),
-    author='Waqas Younas, Zohaib Ijaz, Ismael de Esteban',
-    author_email='waqas.younas@gmail.com, mzohaib.qc@gmail.com, ismael.deesteban@gmail.com',
-    url='https://github.com/gettalent/one-signal-python-sdk',
-    keywords=['onesignal', 'onesignalsdk'],
-    install_requires=[
-        'requests',
-        'pytest'
-    ],
+    author="Waqas Younas, Zohaib Ijaz, Ismael de Esteban",
+    author_email="waqas.younas@gmail.com, mzohaib.qc@gmail.com, ismael.deesteban@gmail.com",
+    url="https://github.com/gettalent/one-signal-python-sdk",
+    keywords=["onesignal", "onesignalsdk"],
+    install_requires=["requests", "pytest"],
```

Comment:
I would drop pytest from here, as it's only probably CI related requirement I guess 👍